### PR TITLE
Increase heap

### PR DESCRIPTION
**Pull request by jenswi-linaro**

### CODE REVIEW

File: core/arch/arm/arm.mk [hunk 1]
```diff
@@ -1,4 +1,7 @@
 CFG_LTC_OPTEE_THREAD ?= y
+# Size of emulated trustzone protected SRAM, 300 kB.
```
Comment (jbech-linaro) [inline core/arch/arm/arm.mk]: Not that I care that much, but `s/trustzone/TrustZone/`

**Comment by jbech-linaro** [comment]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Comment by jforissier** [comment]
There is a typo in the first commit description (`TZSRAM_SIZZE`). Also, don't forget to update `plat-imx` when rebasing on top of master. Other than that:
```
Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>
Tested-by: Jerome Forissier <jerome.forissier@linaro.org> (Hikey pager)
```

**Comment by jenswi-linaro** [comment]
Updated, tags applied.

**Pull request merged**